src/3p-frame.js

```diff
@@ -196,6 +198,30 @@ function getDefaultBootstrapBaseUrl(parentWindow) {
 }
 
 /**
+ * Sub domain on which the 3p iframe will be hosted.
+ * Because we only calculate the URL once per page, this function is only
+ * called once and hence all frames on a page use the same URL.
+ * @return {string}
+ */
+export function getSubDomain(win) {
```

Comment:
Is it exported for testing only? I'm a bit concerned that it's documented as a called-once, but there's no code to ensure it doesn't yield singular values. If it's private/testing-only - that's fine. But then i'd also name it as such.